contributed_definitions/nyaml/NXcalibration.yaml

```diff
@@ -21,6 +40,9 @@ NXcalibration(NXobject):
     doc: |
       A digital persistent identifier (e.g., DOI, ISO standard) referring to a detailed description of a
       calibration method but no actual calibration data.
+    calibration_type(NX_CHAR):
```

Comment:
No this is not a duplication. The intention is to add `calibration_type` to the method. In case there are several methods and each method has different types.

Reply:
But each calibration only has one `method` and one `type`, why does the method need to have a `calibration_type` as well?

Reply:
Yeah, probably you are right. Actually, I thought myself in more extended way.